cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -1320,35 +1320,26 @@ func TestControllerManagerAliases(t *testing.T) {
 }
 
 func TestWatchListClientFlagUsage(t *testing.T) {
-	assertWatchListClientFeatureDefaultValue(t)
```

Comment:
since we are going to change the value of the FG in `NewKubeControllerManagerOptions`. i slightly changed the tests so that we validate the default value for KCM after creating the options.